Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -298,25 +312,73 @@ theorem degrees_esymm [Nontrivial R] (n : ℕ) (hpos : 0 < n) (hn : n ≤ Fintyp
 
 end ElementarySymmetric
 
+section CompleteHomogeneousSymmetric
+
+open Finset Multiset Sym
+
+variable [DecidableEq σ] [DecidableEq τ]
+
+/-- The `n`th complete homogeneous symmetric `MvPolynomial σ R`. -/
+def hsymm (n : ℕ) : MvPolynomial σ R := ∑ s : Sym σ n, (s.1.map X).prod
+
+lemma hsymm_def {n : ℕ} : hsymm σ R n = ∑ s : Sym σ n, (s.1.map X).prod := rfl
```

Comment:
This one is still around

Reply:
Must have been in one of the reverted commits.